app/javascript/components/batch_update/components/BatchUpdateForm.spec.js

```diff
@@ -107,53 +158,118 @@ describe('with only null genes', () => {
   })
 })
 
-describe('with currently added genes', () => {
+describe('with pending changes', () => {
   let wrapper, component
 
   beforeEach(() => {
     wrapper = mount(<BatchUpdateForm {...props} />)
     component = wrapper.instance()
-    component.setState({
-      geneValues: {
-        Kawaii: 0,
-        Animals: 100,
-      },
-    })
-    wrapper.update()
   })
 
-  it('enables the "Queue" button', () => {
-    expect(wrapper.find('Button.queue button').prop('disabled')).toEqual(false)
-  })
+  const renderState = state => {
+    component.setState(state)
+    wrapper.update()
+  }
 
-  it('renders the current genes', () => {
-    expect(wrapper.find(GeneInput).length).toEqual(2)
-    expect(wrapper.text()).toMatch('Kawaii')
-    expect(wrapper.text()).toMatch('Animals')
+  it('the "Queue" button is disabled prior to any pending changes', () => {
```

Comment:
I agree with the change on L161 but I'd leave the wording of L174 as-is. 

We should strive to make these "it" statements read grammatically in Jest, RSpec, etc — this goes with the grain of the describe/it language for behavior specs, and makes for more readable test runs:

```
BatchUpdateForm
  with pending changes
    the "Queue" button is disabled prior to any pending changes
```

versus 

```
BatchUpdateForm
  with pending changes
    enables the "Queue" button
```

Reply:
Gotcha, this test actually shouldn't be here it was just asserting a baseline _without_ pending changes, which is covered elsewhere. I've just removed it, we duplicate the logic elsewhere.